packages/backend/src/middlewares/validateRequest.ts

```
@@ -0,0 +1,24 @@
+import { NextFunction, Request, Response } from 'express';
+import { ZodError, ZodTypeAny } from 'zod';
+
+/**
+ * Middleware to validate request body (default) with Zod schema
+ * @param schema ZodTypeAny schema to validate against
+ * Attaches parsed data to req.validated
+ */
+export function validateRequest(schema: ZodTypeAny) {
+  return (req: Request, res: Response, next: NextFunction): Response | void => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)